### PR TITLE
Nemo/v2/hookup redux

### PR DESCRIPTION
**Pull request by nemodjuric**

### CODE REVIEW

File: client/app/certification/DocumentsCheckTable.jsx [hunk 1]
```diff
@@ -40,14 +40,15 @@ const DocumentsCheckTable = (props) => {
...
         </tr>
-        {props.ssocDates.map((ssocDate, index) =>
-          <tr id={`ssoc-${index + 1}-match`} key={index}>
-            <td><FoundIcon/></td>
-            <td>SSOC {index + 1}</td>
-            <td>{ssocDate}</td>
-            <td>{ssocDate}</td>
-          </tr>
-        )}
+        { typeof props.ssocDates !== 'undefined' && props.ssocDates.length > 0 && props.ssocDates.map((ssocDate, index) =>
```
Comment (amprokop) [inline client/app/certification/DocumentsCheckTable.jsx]: this line looks too long and that it'll fail the linter

try
```
{ typeof props.ssocDates !== 'undefined' && props.ssocDates.length > 0 
   && props.ssocDates.map((ssocDate, index) =>
```

**Comment by amprokop** [comment]
hey @nemodjuric — it looks like the same two tests fail here, one with a `Poltergeist:DeadClient`

you could try adding `retry: 5` to the test that's failing— see: https://github.com/department-of-veterans-affairs/caseflow/pull/1389 — and see if that clears it up.

**Comment by amprokop** [comment]
@nemodjuric — please add a description and screenshots to this PR as well — this should be standard practice for every PR that includes UI changes.

File: client/app/certification/DocumentsCheck.jsx [hunk 2]
```diff
@@ -57,11 +54,9 @@ const mapStateToProps = (state) => {
     nodDate: state.nodDate,
     socMatch: state.socMatch,
     socDate: state.socDate,
+    match: state.match
```
Comment (amprokop) [inline client/app/certification/DocumentsCheck.jsx]: don't need this line

Comment (nemodjuric) [inline client/app/certification/DocumentsCheck.jsx]: Yes. There was confusion in the code with 'match' and something that was supposed to be 'documentsMatch'. I added latter to to_hash in certification.rb and refactored the code.

**Pull request merged**